info.femm.unofficial-manifest.yml

```diff
@@ -0,0 +1,233 @@
+app-id: info.femm.unofficial
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: femm.sh
+build-options:
+  env:
+    - "WINEPREFIX": "/var/data/.local/share/femm42-flatpak"
+    - "WINEDLLOVERRIDES": "mscoree,mshtml="
+    - "V": "1"
+
+#rename-desktop-file: wine.desktop
+
+# Support 32-bit at runtime
+add-extensions:
+  org.freedesktop.LinuxAudio.Plugins:
+    directory: extensions/Plugins
+    versions: '21.08'
+    add-ld-path: lib
+    merge-dirs: lxvst;vst3
+    subdirectories: true
+    no-autodownload: true
+
+  org.freedesktop.Platform.Compat.i386:
+    directory: lib/i386-linux-gnu
+    version: '21.08'
+
+# Support 32-bit at buildtime
+sdk-extensions:
+  - org.freedesktop.Sdk.Compat.i386
+  - org.freedesktop.Sdk.Extension.toolchain-i386
+
+finish-args:
+  - --env=WINEDEBUG=-all
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --socket=pulseaudio
+  - --allow=multiarch
+  - --allow=devel
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.gnome.Mutter.DisplayConfig
+  - --talk-name=org.freedesktop.Flatpak
+  - --filesystem=host
+  - --filesystem=~/.local/share/femm42-flatpak/:create
+  - --device=all
+  - --device=dri
+  # wineprefixes are created here
+  - --persist=~/.local/share/femm42-flatpak/
+  # Support 32-bit runtime
+  # Testing for Driver San andreas- --env=LD_LIBRARY_PATH=/app/lib:/app/lib:/app/lib32:/app/lib64:/app/lib/i386-linux-gnu:/app/lib/wine:/app/lib64/wine:$(pwd)
+  - --env=LD_LIBRARY_PATH=/app/lib:/app/lib:/app/lib32:/app/lib64:/app/lib/i386-linux-gnu:/app/lib/wine:/app/lib64/wine
+
+modules:
+
+  # Create 32-bit directories
+  - name: compat
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/lib/i386-linux-gnu 
+      - mkdir -p /app/lib/debug/lib/i386-linux-gnu
+      - mkdir -p /app/lib/udev/rules.d
+      - mkdir -p /app/extensions/Plugins
+      - mkdir -p /app/lib64
+      - mkdir -p /app/lib32
+
+
+# build wow64              
+  - name: org.winehq.wine_64bit-5.0.5
+    builddir: true
+    build-options:
+      prefix: /app
+      prepend-path: /app/bin
+      ldflags: -L/app/lib64 -L/app/lib -L/app/lib32
+      append-path: /usr/lib/sdk/toolchain-i386/bin
+      env:
+        CC:  i686-unknown-linux-gnu-gcc
+        CXX: i686-unknown-linux-gnu-g++
+      libdir: /app/lib32
+      strip: true
+      no-debuginfo: true
+      
+    make-install-args:
+      - LDCONFIG=/bin/true
+      - STRIP=/bin/true
+      - UPDATE_DESKTOP_DATABASE=/bin/true
+    buildsystem: simple
+    build-commands:
```

Comment:
You don't need to (and should not) do the complex WoW64 build. Please see [this thread](https://github.com/flathub/flathub/pull/2586#discussion_r730251980) for a similar sitation (and looks like you've copied the improper wine module from the same source).

Reply:
This my first PR. If I commit the new files, are you able to see all modifications?

Reply:
Well, if I understood correctly what you mean, the files are in a git repo, so we should be able to see the changes, unless you force-push them.